.github/workflows/prepare_mkdocs.sh

```diff
@@ -38,14 +24,14 @@ title_markdown_file() {
   mv "$1.fixed" "$1"
 }
 
+set +x
+for MARKDOWN_FILE in $(find docs/4.x/ -name '*.md'); do
```

Comment:
4.x? Caught cheating on the test from OkHttp.

Reply:
LOL yup good one